source/fundamentals/authentication/mechanisms.txt

```diff
@@ -139,7 +139,7 @@ To connect to a MongoDB instance with ``MONGODB-AWS`` authentication
 enabled, specify the ``MONGODB-AWS`` authentication mechanism and pass
 your ``AWS_ACCESS_KEY_ID`` and ``AWS_SECRET_ACCESS_KEY`` credentials to the
 driver when you attempt to connect. If your AWS login requires a session
-token, you must include your ``AWS_SESSION_TOKEN`` as well.
+token, you must include your ``AWS_SESSION_TOKEN`` as well. 
```

Comment:
Nitpick: no reason to add a trailing space (I usually use a hotkey I set up in my text editor to strip all trailing spaces) in rst.

Reply:
accepted